src/snowflake/cli/api/sql_execution.py

```diff
@@ -90,6 +89,12 @@ def execute_query(self, query: str, **kwargs):
     def execute_queries(self, queries: str, **kwargs):
```

Comment:
can we backfill the doc comments too while we're here for all public methods in the contract?